README.md

```diff
@@ -28,21 +33,16 @@ pip install nm-vllm[sparse] --extra-index-url https://pypi.neuralmagic.com/simpl
 
 ### Docker
 
-The `nm-vllm-ent` [container registry](https://github.com/neuralmagic/nm-vllm-certs/pkgs/container/nm-vllm-ent) includes premade docker images.
+The `nm-vllm-certs` [container registry](https://github.com/neuralmagic/nm-vllm-certs/pkgs/container/nm-vllm-certs) includes premade docker images.
```

Comment:
We use nm-vllm-ent as the docker image name so the old URL is actually correct, please change this back. The docker image is published in the nm-vllm-certs repo.

Reply:
good catch! i'll revert.